internal/transport/keepalive_test.go

```diff
@@ -370,8 +337,8 @@ func (s) TestKeepaliveClientStaysHealthyWithResponsiveServer(t *testing.T) {
 		normal,
 		ConnectOptions{
 			KeepaliveParams: keepalive.ClientParameters{
-				Time:                1 * time.Second,
-				Timeout:             1 * time.Second,
+				Time:                55 * time.Millisecond,
+				Timeout:             10 * time.Millisecond,
```

Comment:
Leave this set to a higher value, please, since we're not expecting it to be hit.  1s?